clients/drcachesim/tracer/raw2trace.h

```diff
@@ -253,6 +262,11 @@ struct instr_summary_t final {
     static const int kIsFlushMask = 0x0008;
     static const int kIsCtiMask = 0x0010;
 
+    // kIsAarch64DcZvaMask is available in non-AArch64 builds too, but it's intended for
```

Comment:
nit: We have x86 builds (i.e., executes on x86) that target AArch64 traces.  Maybe best to reword here to clarify.

Reply:
Done.